lib/Dialect/CGGI/Transforms/CMakeLists.txt

```diff
@@ -1,3 +1,47 @@
+# Since this defines two passes in one *.td file,
```

Comment:
is this something we should mark as a HEIR style thing? i can file a follow-up issue that we should probably make a Developer Best Practices / Style guide.

Reply:
The transforms (especially the non-dialect-associated ones in `lib/Transforms`) are a bit of a wild west, even in Bazel-land. I do think we should set-up some kind of "HEIR style" - is there a way to have something similar to `add_heir_pass` (in `cmake/AddHEIR.cmake`) for bazel, too? That way, the style becomes self-enforcing ;)

Reply:
Done, I left the comments in heir-opt, which needs nearly all of them anway, but removed them from heir-translate and heir-lsp

Reply:
> is there a way to have something similar to add_heir_pass

I think so! There's bazel macros that can generate the library rules for you (MLIR uses this for the custom tablegen builds), so we could actually create a bazel macro for a heir_dialect_library and so on. I think it would remove some of the boilerplate. I'll file an issue about it for later

Reply:
> for bazel, too? That way, the style becomes self-enforcing ;)

The Google internal processes force a bazel style guide, and Asra and I regularly patch PRs internally to keep it up to standard. I doubt there's a ton of benefit going beyond that for MLIR specific stuff, but I'm open to suggestions. File an issue.